providers/linkedin/handlers.go

```diff
@@ -37,7 +36,13 @@ func (c *Connector) buildSingleObjectMetadataRequest(ctx context.Context, object
 	}
 
 	if ObjectsWithSearchQueryParam.Has(objectName) {
-		url.WithQueryParam("q", "search")
+		if objectName == "dmpSegments" {
+			url.WithQueryParam("q", "account")
+
```

Comment:
add link to https://learn.microsoft.com/en-us/linkedin/marketing/matched-audiences/create-and-manage-segments?view=li-lms-2025-08&tabs=http#find-dmp-segments-by-account, since this is unusual